### PR TITLE
[docs] documentation of the token expiration and scavenging

### PR DESCRIPTION
**Pull request by dacrome**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -100,20 +104,41 @@ The name of the extension field for the specific field.
...
 
 The needed scopes of the self administration client:
 For the standard example installation you can put in 'GET,POST,PUT,PATCH,DELETE'
 -> Actually the scopes are just granulated as the known HTTP-Methods and effective for all resources<br />
 More information on scopes which are configured in the OSIAM server, please click [here](https://github.com/osiam/server/wiki/api_documentation#scopes)
 
+#### org.osiam.addon-self-administration.registration.activation-token-timeout
+
+Specify how long the activation token is valid before the user has to be
+activate a new one. Also the same property to configure the scavenger task
```
Comment (tkrille) [inline docs/configuration.md]: "bofore the user has to request a new one"?

File: docs/configuration.md [hunk 2]
```diff
@@ -100,20 +104,41 @@ The name of the extension field for the specific field.
...
+and it's cleanup of the expired or invalid activation tokens. Default: 24h
```
Comment (tkrille) [inline docs/configuration.md]: maybe add some examples of valid values?

File: docs/configuration.md [hunk 3]
```diff
@@ -100,20 +104,41 @@ The name of the extension field for the specific field.
...
+
+#### org.osiam.addon-self-administration.change-email.confirmation-token-timeout
+
+Specify how long the confirmation token is valid before the user has to be
+activate a new one. Also the same property to configure the scavenger task
+and it's cleanup of the expired or invalid confirmation tokens. Default: 24h
```
Comment (tkrille) [inline docs/configuration.md]: same as above

File: docs/configuration.md [hunk 4]
```diff
@@ -100,20 +104,41 @@ The name of the extension field for the specific field.
...
+
+#### org.osiam.addon-self-administration.lost-password.one-time-password-timeout
+
+Specify how long the one time password is valid before the user has to be
+activate a new one. Also the same property to configure the scavenger task
+and it's cleanup of the expired or invalid one time passwords. Default: 24h
```
Comment (tkrille) [inline docs/configuration.md]: same as above

File: docs/configuration.md (hunk 2 above)
Comment (tkrille) [inline docs/configuration.md]: IMHO, the relation to the scavenger is not clearly expressed.

File: docs/configuration.md [hunk 5]
```diff
@@ -10,11 +10,15 @@
     - [org.osiam.auth.client.id](#orgosiamauthclientid)
     - [org.osiam.auth.client.secret](#orgosiamauthclientsecret)
     - [org.osiam.auth.client.scope](#orgosiamauthclientscope)
+    - [org.osiam.addon-self-administration.registration.activation-token-timeout](#orgosiamaddon-self-administrationregistrationactivation-token-timeout)
+    - [org.osiam.addon-self-administration.change-email.confirmation-token-timeout](#orgosiamaddon-self-administrationchange-emailconfirmation-token-timeout)
+    - [org.osiam.addon-self-administration.lost-password.one-time-password-timeout](#orgosiamaddon-self-administrationlost-passwordone-time-password-timeout)
+    - [org.osiam.addon-self-administration.one-time-token-scavenger.enabled](#orgosiamaddon-self-administrationone-time-token-scavengerenabled)
```
Comment (tkrille) [inline docs/configuration.md]: the links don't work :(

Comment (dacrome) [inline docs/configuration.md]: done.

**Pull request merged**